Packs/CheckPointSandBlast/Integrations/CheckPointSandBlast/CheckPointSandBlast.py

```diff
@@ -867,7 +873,7 @@ def get_file_indicator(file_hash: str, hash_type: str, response: Dict[str, Any])
         indicator=file_hash,
         indicator_type=DBotScoreType.FILE,
         integration_name='CheckPointSandBlast',
-        reliability=DBotScoreReliability.C,
+        reliability=demisto.params().get('integrationReliability'),
```

Comment:
It would be better to save it in the client and when calling `get_file_indicator` from `get_file_indicator` pass the reliability too.

Reply:
done